internal/repository/index.go

```diff
@@ -223,22 +181,6 @@ func (idx *Index) Has(bh restic.BlobHandle) bool {
 	return idx.byType[bh.Type].get(bh.ID) != nil
 }
 
-// LookupSize returns the length of the plaintext content of the blob with the
```

Comment:
Are you sure we don't want to keep a separate path for looking up sizes? It short-circuits when there are duplicates and it doesn't allocate.

Reply:
I'm not too happy that `LookupSize` has to reimplement `Blob.DataLength()`, but I'm not settled on getting rid of that function or not.

Reply:
I've removed the commit for now.